aleph0/README.md

```diff
@@ -24,3 +24,31 @@ CONSOLA_LEVEL=4 bun src/index.ts gen "create a prisma schema file with a blog mo
 ```bash
 CONSOLA_LEVEL=4 bun run src/index.ts gen "add a file named agam.ts in the app/ folder" -p ../../examples/next -rd false
 ```
+
+## Snippets in aleph0
+
+### Overview
+
+Snippets are key components in aleph0, providing valuable context to the AI. They offer insights into how various frameworks are used and demonstrate our coding conventions and patterns.
+
+### Snippet Structure
+
+- **Location**: Snippets are located in the `snippets` folder.
+- **Types of Snippets**:
+  - **General Technology Snippets**: These explain aspects of a framework. For instance, `page.tsx` in Next.js.
```

Comment:
`For instance, `page.tsx` in Next.js.` - not sure this explains anything

Reply:
Better?